src/components/ClaimReview/form/DynamicReviewTaskForm.tsx

```diff
@@ -155,10 +161,20 @@ const DynamicReviewTaskForm = ({ data_hash, personality, claim }) => {
     };
 
     const handleSendEvent = async (event, data = null) => {
+        const editorContent = editorRef.current;
         if (!data) {
             data = getValues();
         }
 
+        if (editorContent) {
+            setCollabEditorError(null);
+            if (!editorContent.html.includes("<a")) {
```

Comment:
suggestion (blocking): you should check for a attribute that identify the HTML element as a "Aletheia Source"